### PR TITLE
Fission/mobile data collection tool blog

### PR DESCRIPTION
**Pull request by hemanthReddy06**

### CODE REVIEW

File: dimagi/pages/urls/redirect.py [hunk 1]
```diff
@@ -173,4 +173,7 @@
     url(r'^an-asha-calls-me-and-tells-me-stories-of-change/$', redirect.blog('an-asha-calls-me-and-tells-me-stories-of-change')),
     url(r'^africare-dimagi-project-selected-as-a-slb-finalist/$', redirect.blog('africare-dimagi-project-selected-as-a-slb-finalist')),
     url(r'^5-div-grantees-to-be-showcased-at-the-unga-mdg-countdown/$', redirect.blog('5-div-grantees-to-be-showcased-at-the-unga-mdg-countdown')),
+    url(r'^blog/mobile-data-collection-standards/$', redirect.blog('mobile-data-collection-mobile-survey-design')),
```
Comment (biyeun) [inline dimagi/pages/urls/redirect.py]: this doesn't seem like it belongs in this PR

**Comment by biyeun** [comment]
@hemanthReddy06 like with the other pull request, you have two commits in the beginning that do not fit here. can you please rebase master into this branch and force push this branch back up so that the commit history is clean?

File: dimagi/pages/templates/sections/blog/mobile_blog_single/blog_why_surveys.html [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+                Why mobile surveys instead of paper?
+              {% endblocktrans %}
+            </h2>
+            <p>
+              {% blocktrans %}
+                While it may be the standard method of data collection at many organizations, there are significant issues associated with a
+                paper-based data approach.
+              {% endblocktrans %}
+            </p>
+            <p>
```
Comment (biyeun) [inline dimagi/pages/templates/sections/blog/mobile_blog_single/blog_why_surveys.html]: this should be an `<h3>` tag, remove the `<strong>` tag

File: dimagi/pages/templates/sections/blog/mobile_blog_single/blog_requirement_affect.html [hunk 3]
```diff
@@ -0,0 +1,61 @@
+{% load i18n %}
+{% load dimagidotcom %}
+
+<section class="border-top blue-theme content-width-medium">
```
[the remark this answers is not in the document]
Comment (hemanthReddy06) [reply dimagi/pages/templates/sections/blog/mobile_blog_single/blog_requirement_affect.html]: Hi, @biyeun I've added nothing to this section it is taking automatically. I wrote the same as remaining sections.

File: dimagi/pages/static/style/pages/about.scss [hunk 4]
```diff
@@ -2,115 +2,105 @@
 @import 'app/variables/colors';
 @import 'app/variables/sizes';
```
Comment (biyeun) [inline dimagi/pages/static/style/pages/about.scss]: hey it looks like there are whitespace changes here...can you please undo this? 2 spaces in css. also `about.scss` shouldn't be touched in this PR

File: dimagi/pages/static/style/pages/blog.scss [hunk 5]
```diff
@@ -2,73 +2,62 @@
...
-
 .blog-page {
-  @include make-bg(hero, '../../pages/blog/hero');
+    @include make-bg(hero, '../../pages/blog/hero');
 }
 
 .post-thumbnail {
-  @include background-placeholder();
-  background-position: center;
+    @include background-placeholder();
```
Comment (biyeun) [inline dimagi/pages/static/style/pages/blog.scss]: 2 space indents in scss please!

File: dimagi/pages/static/style/pages/mdc_tool_blog.scss [hunk 6]
```diff
@@ -0,0 +1,64 @@
+@import 'app/mixins/background';
+@import 'app/variables/colors';
+@import 'app/variables/sizes';
+@import 'app/variables/type';
+#how-to-choose {
+    @include make-bg(epic-cover, '../../pages/blog/mobiledata_blog_page/how_to_choose/hero');
```
Comment (biyeun) [inline dimagi/pages/static/style/pages/mdc_tool_blog.scss]: two space indents in scss please

File: dimagi/pages/static/style/pages/blog.scss [hunk 7]
```diff
@@ -63,12 +59,6 @@ a.post {
...
 .powerful-data {
   $_base-height: 175px;
-  $_height-by-size: (
-    extra-small: 205px,
-    small: 270px,
-    medium: 210px,
-    large: 265px,
-    extra-large: 320px,
-  );
+  $_height-by-size: ( extra-small: 205px, small: 270px, medium: 210px, large: 265px, extra-large: 320px, );
```
Comment (biyeun) [inline dimagi/pages/static/style/pages/blog.scss]: just fYI. the best practices for our `scss` code, and this is even present in the `scss` lint file that you should link with your editor, is that EACH property must be on a separate line. Please change this back to what it was. Thanks

File: dimagi/pages/static/style/pages/blog.scss [hunk 8]
```diff
@@ -53,7 +50,6 @@ a.post {
   img {
     margin-bottom: $space-small;
   }
-
```
Comment (biyeun) [inline dimagi/pages/static/style/pages/blog.scss]: why was this space deleted? there should be a space between class definitions in `scss`. this is in the lint file in the root of the project

File: dimagi/pages/static/style/pages/blog.scss [hunk 9]
```diff
@@ -15,7 +15,6 @@
 a.post {
   color: $color-text !important; // scss-lint:disable ImportantRule
   font-weight: $font-weight-regular;
-
```
Comment (biyeun) [inline dimagi/pages/static/style/pages/blog.scss]: please insert the line between definitions back. don't delete them. there shouldn't be any change in the `blog.scss` file

File: dimagi/pages/templates/sections/blog/mobile_blog_single/blog_affect.html [hunk 10]
```diff
@@ -0,0 +1,48 @@
+{% load i18n %}
+{% load dimagidotcom %}
+
+<section class="blog-build blue-theme pad-top-base">
+  <div class="epic-content content content-width-medium pad-bottom-small-at-medium">
+
+      <div class="copy">
```
Comment (biyeun) [inline dimagi/pages/templates/sections/blog/mobile_blog_single/blog_affect.html]: 4 space indent when there should be 2

File: dimagi/pages/templates/sections/blog/mobile_blog_single/blog_build.html [hunk 11]
```diff
@@ -0,0 +1,38 @@
...
+              {% endblocktrans %}
+            </p>
+          </div>{# .column #}
+
+          <div class="grid-2-at-medium">
+            {% include 'sections/blog/blog_build/track.html' %}
+            {% include 'sections/blog/blog_build/ofline.html' %}
+            {% include 'sections/blog/blog_build/endusers.html' %}
+            {% include 'sections/blog/blog_build/scale.html' %}
+        </div>
```
Comment (biyeun) [inline dimagi/pages/templates/sections/blog/mobile_blog_single/blog_build.html]: uneven indentation

File: dimagi/pages/templates/sections/blog/mobile_blog_single/blog_learnmore.html [hunk 12]
```diff
@@ -0,0 +1,32 @@
...
+        {% endblocktrans %}
+      </p>
+      <p class="align-center">
+      <a class="button button-accent-gray {% click_body 'mobile-data-collection' %}"
+         href="{% url 'mobile_data_collection' %}">
+        {% blocktrans %}
+          Learn How
+        {% endblocktrans %}
+      </a>
+    </p>
```
Comment (biyeun) [inline dimagi/pages/templates/sections/blog/mobile_blog_single/blog_learnmore.html]: uneven indentation

File: dimagi/pages/templates/sections/blog/mobile_blog_single/blog_why_surveys.html [hunk 13]
```diff
@@ -0,0 +1,41 @@
+{% load i18n %}
+{% load dimagidotcom %}
+
+<section class="blue-theme">
+  <div class="content epic-content content-width-medium pad-top-medium">
+
+      <div class="copy">
```
Comment (biyeun) [inline dimagi/pages/templates/sections/blog/mobile_blog_single/blog_why_surveys.html]: 4 space indentation when there should be 2

File: dimagi/pages/templates/svg/blog/icon/mobile_surveys/slow_report.html [hunk 14]
```diff
@@ -0,0 +1,7 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 68 68" width="68px" height="68px">
+  <path d="M44.1,50.8H16.8a10.8,10.8,0,0,1,0-21.6h1a13.33,13.33,0,0,1-.3-2.8,16.2,16.2,0,0,1,32.2-2.5A13.45,13.45,0,0,1,61.1,37.2a13.09,13.09,0,0,1-1.2,5.6" transform="translate(-5 -9.2)" fill="none" stroke="#004fbc" stroke-linecap="round" stroke-linejoin="round" stroke-width="2"></path>
+  <polyline points="23.4 21.2 28.7 16.2 34.2 21.2" fill="none" stroke="#004fbc" stroke-linecap="round" stroke-linejoin="round" stroke-width="2"></polyline>
+  <line x1="28.7" y1="16.2" x2="28.7" y2="28.8" fill="none" stroke="#004fbc" stroke-linecap="round" stroke-linejoin="round" stroke-width="2"></line>
+  <circle cx="47.9" cy="39.4" r="9.1" fill="none" stroke="#004fbc" stroke-linecap="round" stroke-linejoin="round" stroke-width="2"></circle>
+  <polyline points="47.7 35.8 47.7 39.4 50.9 41.2" fill="none" stroke="#004fbc" stroke-linecap="round" stroke-linejoin="round" stroke-width="2"></polyline>
+</svg>
```
Comment (biyeun) [inline dimagi/pages/templates/svg/blog/icon/mobile_surveys/slow_report.html]: newline at the end of all of these svg files plese

**Pull request merged**